airflow/executors/kubernetes_executor.py

```diff
@@ -772,7 +773,15 @@ def try_adopt_task_instances(self, tis: Sequence[TaskInstance]) -> Sequence[Task
         kube_client: client.CoreV1Api = self.kube_client
         for scheduler_job_id in scheduler_job_ids:
             scheduler_job_id = pod_generator.make_safe_label_value(str(scheduler_job_id))
-            query_kwargs = {"label_selector": f"airflow-worker={scheduler_job_id}"}
+            # We will look for any pods owned by the no-longer-running scheduler,
```

Comment:
do we know that `scheduler_job_ids` are all not running?

Reply:
Yes, if you look up a few lines, we build that from the TI's SchedulerJob asks us to try and adopt. And those are TIs are tied to [non-running SchedulerJobs](https://github.com/apache/airflow/blob/14783c63c5d36fcd70afaaebc29c2638073b7e34/airflow/jobs/scheduler_job.py#L1455).